.github/workflows/release.yml

```
@@ -0,0 +1,484 @@
+          name: ip-${{ env.NEW_VERSION }}
+          path: |
+            ${{ env.RELEASE_FILE }}
+            sha256.txt
+            md5.txt
+            RELEASE_NOTES.md
+            ${{ env.LOG_DIR }}/**
+          retention-days: 90
+
+      # Step 9: Create GitHub Draft Pre-Release
+      - name: Create GitHub Draft Pre-Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.10)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)